tower-load/Cargo.toml

```diff
@@ -23,11 +23,11 @@ edition = "2018"
 publish = false
 
 [dependencies]
-futures = "0.1.26"
-log = "0.4.1"
-tokio-timer = "0.2.4"
-tower-service = "0.2.0"
-tower-discover = "0.1.0"
+tracing = "0.1"
```

Comment:
Should moving to `tracing` be done in its own PR?

Reply:
Yeah, I was just doing a drop in replacement.

@hawkw any preference how we approach this?

Reply:
I'm fine with either. I think if we want to replace the unstructured log lines with more idiomatic structured `tracing` diagnostics, _that_ should be done separately; I don't really have a problem with just switching the import in this PR though.

With that said, to avoid changing behavior, we should probably enable `tracing`'s `log` feature, so that we don't break `log` records for users who depend on them. We should probably follow the same strategy as `tower-balance` where we enable that conditionally on a default-on `log` feature flag on this crate, so that users who don't want to emit `log` records can disable it:
https://github.com/tower-rs/tower/blob/793e2e8e94b5ca6c1d12171b3909d78505ab6667/tower-buffer/Cargo.toml#L24-L26